config/Dockerfiles/tests.d/general/01_general

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash -xe
 
 # Verify basic provisioning for all supplied providers
-# distros.exclude: centos7 fedora28 fedora30
+# distros.exclude: centos7 fedora30
```

Comment:
Shouldn't we exclude fedora29 instead of fedora30?

Reply:
Will update it.

Reply:
Deleting 01_general tests since every provider has its own provisioning test. 
It would be a duplicate test.